api/src/main/java/uk/gov/dhsc/htbhf/claimant/controller/ClaimController.java

```diff
@@ -34,7 +34,11 @@
     private final ClaimDTOToClaimConverter converter;
     private final Map<EligibilityStatus, HttpStatus> statusMap = Map.of(
             EligibilityStatus.ELIGIBLE, HttpStatus.CREATED,
-            EligibilityStatus.DUPLICATE, HttpStatus.OK
+            EligibilityStatus.INELIGIBLE, HttpStatus.OK,
```

Comment:
we'll need some unit tests to confirm that we get the right status codes returned

Reply:
ignore me, just seen the test. D'oh